APITienda/Controllers/PersonasController.cs

```diff
@@ -27,30 +25,30 @@ public PersonasController(IPersonaRepository perRepo, IMapper mapper)
         /// Obtener todas las personas.
         /// </summary>
         /// <returns></returns>
-        [HttpGet]
-        public async Task<IActionResult> GetPersonas()
+        [HttpGet(Name = nameof(GetPersonas))]
+        public async Task<ActionResult<List<PersonaDto>>> GetPersonas()
         {
             var listaPersonas = await _perRepo.GetAllAsync();
             var listaPersonasDTO = _mapper.Map<List<PersonaDto>>(listaPersonas);
-            return Ok(listaPersonasDTO);
+            return listaPersonasDTO;
         }
 
         /// <summary>
         /// Crear nueva persona
         /// </summary>
         /// <param name="personaDto"> Esta es la persona que se va a crear</param>
         /// <returns></returns>
-        [HttpPost]
-        public async Task<IActionResult> PostPersona([FromBody] PersonaDto personaDto)
+        [HttpPost(Name = nameof(CrearPersona))]
+        public async Task<ActionResult> CrearPersona(PersonaDto personaDto)
         {
-            if(personaDto == null)
+            if (personaDto is null)
```

Comment:
El operador `is` (pattern matching) funciona como el `==` con el beneficio extra que no puede ser sobrecargado es decir no se puede reasignar el funcionamiento del igual.